### PR TITLE
Navbar restyle

### PR DESCRIPTION
**Pull request by silverbabyyyy**
Restyled sidenav buttons, added user name in toolbar, removed info button

### CODE REVIEW

File: src/app/components/sidenav/sidenav.component.scss [hunk 1]
```diff
@@ -22,7 +21,7 @@
 }
 
 .icon-description{
-    height: 45px;
+    height: 49.9999px;
```
Comment (meda-racasan) [inline src/app/components/sidenav/sidenav.component.scss]: why do you need to use such a value? :)

File: src/styles.scss [hunk 2]
```diff
@@ -88,4 +88,27 @@ label {
...
+}
+ 
+/* Handle */
+::-webkit-scrollbar-thumb {
+    border-radius: 6px;
+    background: $yonder-pink; 
+}
+
+/* Handle hover */
+::-webkit-scrollbar-thumb:hover {
```
Comment (meda-racasan) [inline src/styles.scss:111]: It looks like you have the same style for` ::-webkit-scrollbar-thumb` and `::-webkit-scrollbar-thumb:hover `

File: src/app/components/sidenav/sidenav.component.scss (hunk 1 above)
Comment (silverbabyyyy) [reply src/app/components/sidenav/sidenav.component.scss]: I wanted to line it up perfectly, but I think I found a prettier value that gets probably as close as this one.

File: src/app/components/toolbar/toolbar.component.scss [hunk 3]
```diff
@@ -27,4 +27,24 @@
 .toolbar-text{
     color: variables.$yonder-button-white;
     cursor: default;
+    margin: 0;
+}
+
+.top-right {
+    display: flex;
+    flex-direction: row;
+}
```
Comment (meda-racasan) [inline src/app/components/toolbar/toolbar.component.scss:34]: You could use the flex-container mixin

File: src/app/components/toolbar/toolbar.component.scss [hunk 4]
```diff
@@ -27,4 +27,24 @@
...
+
+.button {
+    display: flex;
```
Comment (meda-racasan) [inline src/app/components/toolbar/toolbar.component.scss]: You could use the flex-container mixin

**Pull request merged**